README.md

```diff
@@ -2,21 +2,26 @@
 
 A minimal recommended setup for an applications using [Cesium](https://cesium.com) with [Webpack](https://webpack.js.org/concepts/).
 
-[![Build Status](https://travis-ci.org/CesiumGS/cesium-webpack-example.svg?branch=using-custom-loader)](https://travis-ci.org/CesiumGS/cesium-webpack-example)
+Jump to the [Webpack 5](./webpack-5/) directory for the most up to date example. We also provide a [Webpack 4](./webpack-4/) example if you are still on the older version.
 
 ## Running this application
 
 ```sh
+# switch to the correspoding webpack-4 or webpack-5 directory
```

Comment:
Does it make sense to include this section at the top level? Or does the `Running this app` section belong in the respective subdirectories?

Reply:
I was back and forth about this but I think I ultimately settled on having a section like this at the top level so it's easier to see when initially coming to the repo. The process for each version is identical so I thought it would be good to keep a copy of the info at the top level too. Maybe it's excessive though?

Reply:
IMO omitting this section in favor of directing readers to the relevant directory will leave less chance for confusion.